package.json

```diff
@@ -13,6 +13,7 @@
     "postinstall": "lerna bootstrap",
     "pretest": "npm run compile",
     "test": "npm run testonly --",
+    "tdd": "npm run compile && mocha --full-trace --timeout 5000 ./test/tests.js",
```

Comment:
Let's keep this as a separate concern outside of this PR since it's not directly related.